src/HashBus.Projector.MostRetweeted/App.cs

```diff
@@ -10,23 +12,34 @@ namespace HashBus.Projector.MostRetweeted
 
     class App
     {
-        public static void Run(string mongoConnectionString, string mongoDBDatabase, string endpointName)
+        public static async Task Run(string mongoConnectionString, string mongoDBDatabase, string endpointName, string applicationAddress)
         {
             var mongoDatabase = new MongoClient(mongoConnectionString).GetDatabase(mongoDBDatabase);
 
-            var busConfiguration = new BusConfiguration();
-            busConfiguration.EndpointName(endpointName);
-            busConfiguration.UseSerialization<JsonSerializer>();
-            busConfiguration.EnableInstallers();
-            busConfiguration.UsePersistence<InMemoryPersistence>();
-            busConfiguration.RegisterComponents(c =>
+            var endpointConfiguration = new EndpointConfiguration(endpointName);
+            endpointConfiguration.UseSerialization<JsonSerializer>();
+            endpointConfiguration.EnableInstallers();
+            endpointConfiguration.UsePersistence<InMemoryPersistence>();
+            endpointConfiguration.RegisterComponents(c =>
                 c.RegisterSingleton<IRepository<string, IEnumerable<Retweetee>>>(
                     new MongoDBListRepository<Retweetee>(mongoDatabase, "most_retweeted__retweetees")));
-            busConfiguration.ApplyMessageConventions();
+            endpointConfiguration.ApplyMessageConventions();
+            endpointConfiguration.ApplyErrorAndAuditQueueSettings();
 
-            using (Bus.Create(busConfiguration).Start())
+            var transportExtensions = endpointConfiguration.UseTransport<MsmqTransport>();
+            var routing = transportExtensions.Routing();

```

Comment:
Please insert a blank line before this line (for consistency with other endpoints).